src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -566,14 +566,14 @@ CharMap =
 #    3gp      3rd Generation Partnership Project            video/3gpp
 #    3g2a     3rd Generation Partnership Project 2 Audio    audio/3gpp2
 #    3g2      3rd Generation Partnership Project 2          video/3gpp2
-#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/aac
+#    aac-lc   Advanced Audio Coding (Low Complexity)        audio/mp4
```

Comment:
@Sami32 Yes, the problem is that there's mime-types for codecs, which is wrong. Only the entries representing formats should have mime-types. Providing wrong information is just confusing to everyone. That's not something this PR has introduced though, so I don't mean that you have to change it, I just had to mention it since you updated one of the meaningless mime-types.

My Onkyo also "identify" both MP4 and ADTS audio files with an AAC stream inside as AAC, but that's not wrong. It's identifying the codec, not the file format.

To try to explain once again - what the above line implies is that a file containing ```aac-lc``` should have a mime-type of ```audio/mp4```. Can't you see that it is wrong? The file could be ADTS, MKV or any other container format that supports AAC. The mime type depends on the format, *not* the codec.